src/content_script/input_tools.js

```diff
@@ -1,235 +1,237 @@
-let SLSettings = {};
+if (!typeof InputTools !== "undefined") {
```

Comment:
this is to make sure this script is run only once right? In this case, I think it'd more clear to create a dedicated variable for that like `alreadyRun`